utils.js

```diff
@@ -1,30 +1,34 @@
-import { memoize } from '@neovici/cosmoz-utils/memoize';
+import { memoooize } from '@neovici/cosmoz-utils/memoize';
 
-const
+const /**
+	 * Get a language sensitive string comparison object.
+	 * @param {string} locale Current locale.
+	 * @returns {object} Collator.
+	 */
+	_makeCollator = (locale) => new Intl.Collator(locale || undefined),
+	makeCollator = memoooize(_makeCollator),
```

Comment:
Do we need to use `memoize` instead ? `memoooize` is mean to be used by function with 3 params ? 
We also should not memoize in the global scope as that memory cannot be released  when we remove the column or omnitable itself.

Reply:
or should I skip it? Just do makeCollator = (locale) => new Intl.Collator(locale || undefined)?

Reply:
its fine with memoize.